frontend/src/app/modules/project/project-create/project-create.component.html

```diff
@@ -0,0 +1,48 @@
+<form (ngSubmit)="save()" #projectSettingForm="ngForm">
+  <h2 class="small-title">
+    Create Project
+  </h2>
+  <div class="form-croup-project">
+    <label for="name-input">
+      <span class="form-control-project">Project name</span>
+      <input type="text" [(ngModel)]="newProject.name" name="name" required autocomplete="off" #name="ngModel"
+        id="name-input" class="form-control" placeholder="My first project">
+    </label>
+    <div class="error" [hidden]="name.errors?.required == null || name.untouched">
```

Comment:
Don't need strick checking. Please, rewrite all similar to: !name.errors?.required || name.untouched